src/index.html

```diff
@@ -7,6 +7,427 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="link-img">
+        <img
+          src="./images/logo.png"
+          class="logo"
+          alt="0"
+        >
+      </a>
+      <nav class="nav">
```

Comment:
Don't forget to add empty lines between multiline sibling blocks of HTML. Check it everywhere
```suggestion
      </a>
      
      <nav class="nav">
```